client/components/slotMaker.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import slot from './slotMaker.css';
```

Comment:
Recommend using styles as the variable instead of slots so it's more apparent that this is a styles object.